includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -873,8 +873,8 @@ protected function prepare_payment_information( $order ) {
 			$payment_information->must_save_payment_method_to_store();
 		}
 
-		if ( $this->platform_checkout_util->should_save_platform_customer() ) {
-			do_action( 'woocommerce_payments_save_user_in_platform_checkout' );
+		if ( $this->woopay_util->should_save_platform_customer() ) {
+			do_action( 'woocommerce_payments_save_user_in_woopay' );
 			$payment_information->must_save_payment_method_to_platform();
```

Comment:
I'm curious why we didn't update this method definition to be
```suggestion
			$payment_information->must_save_payment_method_to_woopay();
```